Add Strimzi Metrics Reporter to documentation & update CHANGELOG.md (#8)
OwenCorrigan76/strimzi-kafka-operator: CHANGELOG.md

* Support for MirrorMaker 1 has been removed
* Added support to configure `dnsPolicy` and `dnsConfig` using the `template` sections.
* Store Kafka node certificates in separate Secrets, one Secret per pod.

> * Support for Strimzi Metrics Reporter for Kafka has been added.

Reviewer: ```suggestion
* Added support for Strimzi Metrics Reporter to the Kafka component.
```

Reviewer: Consistency is always appreciated ;)

Author: Of course. Silly of me to miss that. Fixed now.
